### ISSUE TITLE
Fix variable mismatches caused by this bug: https://github.com/scratchfoundation/scratch-vm/issues/5024

### ISSUE THREAD

**Issue by H-A-M-G-E-R** [TurboWarp/sb3fix#33]

**Comment by GarboMuffin** [comment]
Please don't report bugs to Scratch if you didn't even test them in Scratch...

**Comment by H-A-M-G-E-R** [comment]
i only tested on turbowarp

**Comment by GarboMuffin** [comment]
Yes. So don't report it to Scratch. You didn't test it in Scratch.

**Comment by H-A-M-G-E-R** [comment]
so i'll immediately test it...

**Comment by H-A-M-G-E-R** [comment]
and it does the same thing

**Comment by aliasfoxkde** [comment]
This was a legit bug find, related to both Scratch and TurboWarp (fork). Why is the first response from a dev to shame? Instead of actually helping? Or just not saying anything at all? You do realize that Scratch is geared towards novice and children, right?

**Comment by GarboMuffin** [comment]
The conversation that happened above was discussing them reporting a bug to Scratch without actually testing it in Scratch

**Comment by nezoko45-dev** [comment]
help!
MY SHARKS.IO PROJECT IS STALLING FOR NO REASON!

**Comment by nezoko45-dev** [comment]
link: https://turbowarp.org/editor?offscreen&clones=Infinity
